hc/api/tests/test_admin.py

```diff
@@ -5,15 +5,17 @@
 class ApiAdminTestCase(BaseTestCase):
 
     def setUp(self):
+
         super(ApiAdminTestCase, self).setUp()
         self.check = Check.objects.create(user=self.alice, tags="foo bar")
-
+        self.alice.is_staff = True
+        self.alice.is_superuser = True
+        self.alice.save()
         ### Set Alice to be staff and superuser and save her :)
 
     def test_it_shows_channel_list_with_pushbullet(self):
         self.client.login(username="alice@example.org", password="password")
-
         ch = Channel(user=self.alice, kind="pushbullet", value="test-token")
         ch.save()
-
+        self.assertEqual(ch.kind ,"pushbullet")
```

Comment:
Have a space after the comma, in accordance with [pep8](http://pep8.org/#pet-peeves)